src/calendar/header/index.js

```diff
@@ -141,12 +143,18 @@ class CalendarHeader extends Component {
       indicator = <ActivityIndicator color={this.props.theme && this.props.theme.indicatorColor}/>;
     }
 
+    const webProps = Platform.OS === 'web' ? { 'aria-level': this.props.webAriaLevel } : {};
     return (
       <View style={this.props.style}>
         <View style={this.style.header}>
           {leftArrow}
           <View style={{ flexDirection: 'row' }}>
-            <Text allowFontScaling={false} style={this.style.monthText} accessibilityTraits='header'>
+            <Text
+              allowFontScaling={false}
+              style={this.style.monthText}
+              accessibilityTraits='header'
+              {...webProps}
```

Comment:
I prefer this syntax over the spread and empty object: `aria-level={Platform.OS === 'web' ? this.props.webAriaLevel : undefined}`. Can it work?

Reply:
Sure, the webProps was just a clean way to encapsulate it.  But I can make that change

Reply:
However... you will get a react warning about this, since in native `aria-level` is not a valid prop.  That will drive most developers batty.  The current implementation avoids the warning and is the pattern we use in our app.